exercicios/para-casa/entregas/mylena-verspeelt/InvoiceItem.js

```diff
@@ -0,0 +1,53 @@
+  class InvoiceItem {
+    constructor(id, description, quantity, unityPrice) {
+      (this.id = id),
+        (this.description = description),
+        (this.quantity = quantity),
+        (this.unityPrice = unityPrice);
+    }
+  
+    getID() {
+      return this.id;
+    }
+  
+    getQty() {
+      return this.quantity;
+    }
+  
+    setQty(qtde) {
+      const quantidadeAtualizada = this.quantity + qtde;
+      return quantidadeAtualizada;
+    }
+  
+    getUnityPrice() {
+      return this.unityPrice;
+    }
+  
+    setUnityPrice(value) {
+      const zerarValor = this.unityPrice - this.unityPrice;
+      const precoUnitarioAtualizado = zerarValor + value
+      return precoUnitarioAtualizado;
+    }
+  
+    getTotal(quantity) {
```

Comment:
O metodo getTotal não necessita de parametro, visto que ja temos acesso dinamico ao quantity, atraves do this.quantity